libs/components/theme/src/lib/styles/_sizing.scss

```diff
@@ -117,13 +117,13 @@
 }
 
 .sky-margin-stacked-lg {
-  margin-bottom: $sky-space-xl;
+  margin-bottom: $sky-space-md;
 }
 
 .sky-margin-stacked-xl {
   margin-bottom: $sky-space-xl;
 }
 
 .sky-margin-stacked-xxl {
-  margin-bottom: $sky-space-xl;
+  margin-bottom: 40px;
```

Comment:
Did we double check that we don't have any variable for 40px?

Reply:
Correct, there are no variables for 40px